### PR TITLE
Quality of life changes.

### PR DESCRIPTION
**Pull request by EthanStraub**
- Fishing and mining have increased poudrin gains
- Enemies now attack slower (5 seconds per attack, up from 3)
- The dojo master will tell you how much attacks cost when hitting sandbags
- Slimecoin gains from recycling have been buffed (max possible gain is now 1,000,000 up from 100k)
- !look and !survey now provide more visual clarity when showing enemies in a district
- Minimum slime required to cap is now 200k (up from 50k)
- Using !tp when it is not ready for use will now let you know how long you have to wait, in minutes
- Propstanding has now been made exclusive to apartments. To unstand an item, you must visit The Bazaar. The same principle also applies to aquariums.
- You can use !piss on someone to spend 1 liquid sap and crush 3 of their hardened sap
- Soft cooldown on scavenging has been cut in half. (15 seconds, down from 30)
- Added in !crunch as an alias for !crush
- Added in the Bicarbonate soda fountain. Use it to !purify yourself, which lowers your slime down to 0, level down to 1. It cuts your current weaponskill by 75%, however
- Members of Terezi Gang can now spend 1 slimecoin with !fl1pco1n

### CODE REVIEW

File: ewcmd.py [hunk 1]
```diff
@@ -788,8 +819,31 @@ def weather_txt(id_server):
 	mutations = user_data.get_mutations()
 
 	if ewcfg.mutation_id_enlargedbladder in mutations:
-		response = "You unzip your dick and just start pissing all over the goddamn fucking floor. God, you’ve waited so long for this moment, and it’s just as perfect as you could have possibly imagined. You love pissing so much."
+		if cmd.mentions_count == 0:
+			response = "You unzip your dick and just start pissing all over the goddamn fucking floor. God, you’ve waited so long for this moment, and it’s just as perfect as you could have possibly imagined. You love pissing so much."
+		if cmd.mentions_count == 1:
+			target_member = cmd.mentions[0]
+			target_user_data = EwUser(member = target_member)
```
Comment (magicksRampage) [inline ewcmd.py:828]: we should check for corpse status. ghosts don't have hardened sap

File: ewcmd.py [hunk 2]
```diff
@@ -845,7 +899,7 @@ def weather_txt(id_server):
 			
 			ewitem.item_delete(id_item = item.id_item)
 
-			pay = int(random.random() * 10 ** random.randrange(2,5))
+			pay = int(random.random() * 100 ** random.randrange(2,5))
```
Comment (magicksRampage) [inline ewcmd.py]: operator priority! this increases maximum output by a factor of 10^4. increase to randrange(2,6) instead

File: ewcmd.py [hunk 3]
```diff
@@ -1128,6 +1182,32 @@ def item_off(id_item, id_server, item_name = "", is_pushed_off = False):
...
+async def purify(cmd):
+	user_data = EwUser(member=cmd.message.author)
+	
+	if user_data.poi == ewcfg.poi_id_sodafountain:
+		if user_data.life_state == ewcfg.life_state_corpse:
+			response = "You're too ghastly for something like that. Besides, you couldn't even touch the water if you wanted to, it would just phase right through your ghostly form."
+		else:
+			response = "You close your eyes and hold out your hands to the gentle waters of the bicarbonate soda fountain..."
+			
+			user_data.slimelevel = 1
```
Comment (magicksRampage) [inline ewcmd.py]: won't this be a problem with mutations? players will get additional mutations on level-up

File: ewcmd.py [hunk 4]
```diff
@@ -788,8 +819,31 @@ def weather_txt(id_server):
...
+			target_member = cmd.mentions[0]
+			target_user_data = EwUser(member = target_member)
+			
+			if user_data.poi == target_user_data.poi:
+				if user_data.sap < ewcfg.sap_spend_piss:
+					response = "You don't have enough liquid sap to !piss..."
+				else:
+					sap_damage_target = min(ewcfg.sap_crush_piss, target_user_data.hardened_sap)
+					target_user_data.hardened_sap -= sap_damage_target
+					target_user_data.persist()
```
Comment (magicksRampage) [inline ewcmd.py:845]: it won't persist this damage if you piss on yourself

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: ewcmd.py (hunk 3 above)
Comment (magicksRampage) [reply ewcmd.py]: nevermind, you already fixed it

File: ewcmd.py (hunk 2 above)
Comment (EthanStraub) [reply ewcmd.py]: Understood. Dodged a bullet there lol...

**Pull request merged**